### ISSUE TITLE
Question: about the NER dataset

### ISSUE THREAD

**Issue by chapter544** [Babelscape/ner4el#5]
Dear authors,
How did you construct the NER dataset when training the extended NER system? I see that, for each wiki description, there can be multiple entities (either person, loc ...). Will the dataset be noisy? If possible, can you share this dataset?

Thank you very much!

**Comment by sted97** [comment]
Dear @chapter544,

Thanks for your interest. We constructed the training dataset for our NER classifier in a very simple way. We used only the sentences available in the AIDA-YAGO-CoNLL train split (because we put ourselves in a low-resource setting in the paper), and, for each entry of the form _<left_context, mention, right_context, target_entity>_ we associated a NER type to that entry by using our mapping from Wikipedia IDs to NER tags (target_entity is a Wikipedia ID). Then, given _<left_context, mention, right_context>_ as input, we trained our system to predict the previously assigned NER tag.

Notice that we relied on this formulation for the NER task rather than the standard sequence-labeling one (i.e. identify both the span of the entity and the NER class using BIO tags), because in this case we already had the pre-identified entity spans. Hence, this is a simplified version of the NER task that allows us to achieve higher performance in the Entity Disambiguation task.

Hope this helps.

**Comment by chapter544** [comment]
Hi sted97,
Thank you for your explanations. The paper mentioned the expanded NER categories, so my intention is to train a regular BIO style NER tagging system with the expanded tags. From your experience, can you recommend an approach to construct a span NER dataset given your NER DB (in this paper) as a starting point? Many thanks!

**Comment by sted97** [comment]
The easiest way is to take an entity disambiguation dataset annotated with Wikipedia IDs. Then, you can substitute the Wikipedia IDs with the corresponding NER tags using our mapping. Finally, you tokenize the sentences and project the NER labels over the right tokens.

**Comment by chapter544** [comment]
Thank you for your reply @sted97, I will try this idea.
